azext_iot/central/command_map.py

```diff
@@ -51,6 +51,7 @@ def load_central_commands(self, _):
         cmd_group.command(
             "registration-summary", "registration_summary",
         )
+        cmd_group.command("get-credentials", "get_credentials")
```

Comment:
Is this command useful for IoT Central users? Clearly there is value for tests, but want to make sure this makes sense to expose to end users in the overall story for Central.

Reply:
Lets chat about this one offline